sql/core/src/test/resources/sql-tests/results/explain.sql.out

```diff
@@ -523,12 +571,20 @@ Input     : [key#x, val#x]
      
 (8) HashAggregate [codegen id : 1]
 Input: [key#x]
+Keys: []
+Functions: [partial_max(key#x)]
+Results: [max#x]
```

Comment:
This is hard to read because `max` is a buffer attribute. I think we should also print all the buffer attributes.

We can get it by `HashAggregateExec.aggregateBufferAttributes`

Let's also test `avg` which has 2 buffer attributes.

Reply:
Yeah, add `FuncBufferAttrs` to show the buffer attributes. And modified `Subqueries nested` testcase with `avg` function. 
3614ab2e04b91d96e720e1f77a75ca867a217945